coa/pkg/logger/contexts/activitylog-context.go

```diff
@@ -0,0 +1,489 @@
+package contexts
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/valyala/fasthttp"
+)
+
+const (
+	ACTIVITY_HTTP_HEADER_PREFIX string = "X-Activity-"
```

Comment:
This const uses a different casing

Reply:
Switch to use Activity_HttpHeaderPrefix.